nx.json

```
@@ -8,7 +8,20 @@
     },
     "tsconfig.json": "*",
     "nx.json": "*",
-    ".eslintrc.json": "*"
+    ".eslintrc.json": "*",
+    "CONTRIBUTING.md": ["docs"],
+    "libs/cdk/README.md": ["docs"],
+    "libs/cdk/coalescing/README.md": ["docs"],
+    "libs/cdk/coercing/README.md": ["docs"],
+    "libs/cdk/notifications/README.md": ["docs"],
+    "libs/cdk/render-strategies/README.md": ["docs"],
+    "libs/cdk/transformations/README.md": ["docs"],
+    "libs/cdk/zone-configurations/README.md": ["docs"],
+    "libs/cdk/zone-less/README.md": ["docs"],
+    "libs/cdk/template/README.md": ["docs"],
```

issue: Use case-sensitive path. Make sure the other documents listed here are named `README.md` as specified, not `Readme.md` or any other casing.
```suggestion
    "libs/cdk/template/Readme.md": ["docs"],
```

---

issue fixed in [49ad56e](https://github.com/rx-angular/rx-angular/pull/1436/commits/49ad56e4df1466d1034e9a301ccb0993bd4eca02)

---

Reopened until https://github.com/rx-angular/rx-angular/pull/1436#discussion_r994523814 is resolved.